README.md

```diff
@@ -14,7 +16,7 @@ To see a full example, check out the [main.tf](./example/main.tf) file in the ex
 ```hcl
 module "cloud_security" {
   source      = "sourcefuse/arc-security/aws"
-  version     = "0.0.4"
```

Comment:
We can skip the version in the readme as it is difficult to maintain.

Reply:
yes please remove the `version` from here as by default it picks up the latest version from registry